qa/suites/rgw/multifs/omap_limit/10.yaml

```diff
@@ -0,0 +1,5 @@
+overrides:
+  ceph:
+    conf:
+      osd:
+        osd_max_omap_entries_per_request: 10
```

Comment:
edit: that fix isn't correct either, sorry

i've also identified other cls calls with the same problem:
- cls_log_list
- cls_statelog_list
- cls_timeindex_list